src/goto-programs/show_goto_functions.cpp

```diff
@@ -67,6 +68,38 @@ void show_goto_functions(
       bool is_internal=(has_prefix(id2string(it->first), CPROVER_PREFIX) ||
                         it->first==ID__start);
       json_function["isInternal"]=jsont::json_boolean(is_internal);
+
+      if(it->second.body_available())
+      {
+        json_arrayt json_instruction_array=json_arrayt();
+
+        for(const goto_programt::instructiont &instruction :
+          it->second.body.instructions)
+        {
+          json_objectt instruction_entry=json_objectt();
+
+          std::ostringstream instruction_id_builder;
+          instruction_id_builder << instruction.type;
```

Comment:
I've done this in this commit: https://github.com/diffblue/cbmc/pull/448/commits/da7ce4fb43f965358051a0bd0349ffac87abe76d I couldn't find other uses of it, but it is kind of hard to search for them since `<< type` has so many possible meanings! I was a bit disappointed by the `to_string().c_str()` required, would returning `std::string` make more sense in this case?